cla_backend/apps/cla_provider/tests/api/test_csvupload_api.py

```diff
@@ -992,20 +992,6 @@ def test_df_fixed_fee_amount_invalid(self):
             expected_error = u"Row: 1 - The value you have entered exceeds the Fixed Fee"
             self._test_generated_2018_contract_row_validate_fails(override=test_values, expected_error=expected_error)
 
-    @override_settings(CONTRACT_2018_ENABLED=True)
-    def test_df_fixed_fee_zero_amount_as_missing(self):
-        test_values = {
-            "Eligibility Code": u"V",
-            "Matter Type 1": u"HRNT",
-            "Matter Type 2": u"HPRI",
-            "Stage Reached": u"HA",
-            "Fixed Fee Code": u"DF",
-            "Fixed Fee Amount": u"0",
-            "Determination": u"FINI",
-        }
-        expected_error = u"Row: 1 - Fixed Fee Amount must be entered for Fixed Fee Code (DF)"
-        self._test_generated_2018_contract_row_validate_fails(override=test_values, expected_error=expected_error)
```

Comment:
Given that accidentally doing this wrong, by just checking truthiness, is exceptionally easy to do, I'd prefer to have a test case like this but modified to test that it *isn't* counted as missing

Reply:
How do you propose structuring that so it doesn't duplicate `test_df_fixed_fee_amount_is_valid`?

Reply:
Oh, you're quite right!